app/src/main/res/values/strings.xml

```diff
@@ -75,6 +75,7 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <!-- Settings messages -->
     <string name="msg_contact_us">Contact us</string>
     <string name="msg_language">Language</string>
+    <string name="msg_download_my_data">Download My Data</string>
```

Comment:
Do not forget to add this in all strings resource files, and make sure to add `<!-- TODO Translate -->` to them.

Reply:
Added all translations from the internet. I hope this is okay :)

Reply:
Great but make sure to move all those strings to _Settings messages_ instead of _Report_. There were also some files that moved `TODO Translate` from the previous string to the newly added one.

Reply:
@ubarsaiyan Thanks for adding the translations, but it is ok to leave them in English for all languages we have and add a `<!-- TODO Translate -->` for each of them, because our community members can for sure translate them later.

Thank you.